src/query-builder/relation-id/RelationIdLoader.ts

```diff
@@ -143,12 +143,12 @@ export class RelationIdLoader {
                     return Object.keys(mappedColumn).map(key => {
                         const parameterName = key + index;
                         parameters[parameterName] = mappedColumn[key];
-                        return junctionAlias + "." + key + " = :" + parameterName;
+                        return "\"" + junctionAlias + "\"." + key + " = :" + parameterName;
```

Comment:
not all drivers have quotes as escaping character! You need to use `connection.driver.escape` method. Also please attach test so this functionality won't break ever again. Thank you!